Appointment/Appointment/Utils/Managers/Location/LocationManager.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LocationManager.swift
+//  Appointment
+//
+//  Created by hyosung on 2022/12/27.
+//
+
+import CoreLocation
+
+protocol LocationManagable {
+  var authorizationStatus: CLAuthorizationStatus { get }
+  var delegate: LocationManagerDelegate? { get }
+  
+  func requestWhenInUseAuthorization()
+}
+
+protocol LocationManagerDelegate {
+  func didUpdateLocations(_ locations: [CLLocation])
+}
+
+final class LocationManager: NSObject, LocationManagable {
+  static let shared: LocationManagable = LocationManager()
+  
+  private let locationManager: CLLocationManager = {
+    let manager = CLLocationManager()
+    manager.desiredAccuracy = kCLLocationAccuracyBest
+    manager.distanceFilter = kCLDistanceFilterNone
+    return manager
+  }()
+  
+  var delegate: LocationManagerDelegate?
```

Comment:
@DevVenusK 질문이 있습니다!
사용하는곳에서 delegate를 위임받아서 사용하고 사용이 끝나는 시점에서는 다시 nil로 초기화 해야할까요?
```suggestion
  weak var delegate: LocationManagerDelegate?
```
로 사용하는거는 어떻게 생각하세요?

Reply:
엇 좋습니다! 
사실 지금 Rx가 없는 브랜치라서 일단 delegate 로 만들어뒀고, develop 에 머지가 된다면 delegate 가 아니라 observable 을 이용해 리팩토링 하려고 했어욤 ! 
해당건은 수정해서 다시 올려둘게요! 감사합니당!